internal/pkg/flink/internal/store/store_test.go

```diff
@@ -164,16 +174,22 @@ func TestWaitForPendingStatementErrors(t *testing.T) {
 		client:     client,
 		appOptions: &appOptions,
 	}
+	statusDetailMessage := "Test status detail message"
 	statementObj := flinkgatewayv1alpha1.SqlV1alpha1Statement{
 		Status: &flinkgatewayv1alpha1.SqlV1alpha1StatementStatus{
-			Phase: "COMPLETED",
+			Phase:  "COMPLETED",
+			Detail: &statusDetailMessage,
 		},
 	}
 
-	expectedErr := errors.New("couldn't get statement!")
-	client.EXPECT().GetStatement("orgId", "envId", statementName).Return(statementObj, expectedErr).Times(1)
+	returnedError := errors.New("couldn't get statement!")
```

Comment:
We try to keep punctuation out of errors in case they are nested!
```suggestion
	returnedError := errors.New("couldn't get statement")
```

Reply:
Removed!